test/test_rosbag/test/test_bag.py

```diff
@@ -39,6 +39,7 @@
 import sys
 import time
 import unittest
+import tempfile
```

Comment:
Please move this line two up (for alphabetical order of the core modules [PEP8]).